### ISSUE TITLE
debian 11 publish as recommended image

### ISSUE THREAD

**Issue by ad-m** [hyperonecom/h1-images-recommended#83]
Debian 11 "bullseye" released. 

New image expected to be published at: http://cdimage.debian.org/images/cloud/OpenStack/

Daily build available at: http://cdimage.debian.org/images/cloud/bullseye/daily/latest/

Task to do:

* create a debian bullseye repository - packages.hyperone.cloud, packages.rootbox.cloud
* build & publish CLI package
* build new image & make all tests pass - see #82
* publish by FSa new image as recommened 
* verify ability to use upstream cloud-init - I think cloud-init is fresh enough

**Comment by gregorybrzeski** [comment]
packages.hyperone.cloud, packages.rootbox.cloud - these are `websites`

**Comment by ad-m** [comment]
> packages.hyperone.cloud, packages.rootbox.cloud - these are `websites`

yes, content of that repository are managed by script from that repository https://gitlab.com/hyperonecom/rbx-packaging-distro/

I believe that file need to be updated ( https://gitlab.com/hyperonecom/rbx-packaging-distro/-/blob/master/config/distro/debian ) and repositories regenerated.
